tools/pipelines/templates/build-npm-package.yml

```diff
@@ -478,7 +478,14 @@ stages:
 
                   echo "@fluidframework:registry=${{ variables.feed }}" >> ./.npmrc
                   echo "@fluid-experimental:registry=${{ variables.feed }}" >> ./.npmrc
+
+                  # This is confusing, but we are using the .npmrc here to load dependencies from different feeds. These
+                  # scopes must be loaded from the "dev" feed because that is the only place they are published. Ideally
+                  # this logic will be centralized outside of CI in the future.
                   echo "@fluid-internal:registry=${{ variables.devFeed }}" >> ./.npmrc
+                  echo "@fluid-private:registry=${{ variables.devFeed }}" >> ./.npmrc
+
+                  # These scopes must be loaded from different internal feeds beyond the Fluid feeds.
```

Comment:
To clarify, `$(ado-feeds-build)` is just our build feed, and `$(ado-feeds-office)` used to be a feed in the Office org, but we then created one on our org which upstreams the Office one (to simplify our auth story for it). So the comment might be correct for the second one, but I'd argue not for the first one. Maybe update it to just say those scopes are loaded from fixed feeds, instead of pipeline-run-dependent ones? (In the 3 updated yml templates).